src/Bridge/Symfony/DependencyInjection/NelmioAliceExtension.php

```diff
@@ -17,17 +17,29 @@
 use InvalidArgumentException;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
 use Symfony\Component\Finder\Finder;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
 
 /**
  * @private
  */
-final class NelmioAliceExtension extends Extension
+final class NelmioAliceExtension extends Extension implements PrependExtensionInterface
 {
     const SERVICES_DIR = __DIR__.'/../Resources/config';
-    
+
+    public function prepend(ContainerBuilder $container): void
+    {
+        if (isset($container->getExtensions()['framework'])) {
+            $container->prependExtensionConfig('framework', [
```

Comment:
I do not understand that piece, doesn't that mean the bundle _enables_ the property access? I don't think it's a good idea for the bundle to attempt to change the default settings like so. Your error is a better option IMO, it shouldn't be hard for the user to figure it out and would avoid any surprises of having the property access component enabled when though disabled

Reply:
Yes with that, the bundle will enable the property_access. 
It was just to avoid having an error when you require the package.
I'll remove this part then :+1: